.github/workflows/test-wasm.yaml

```diff
@@ -73,5 +60,284 @@ jobs:
         run: |
           source "$HOME/emsdk/emsdk_env.sh"
           emmake make test
+      - name: Pack build directory
+        run: tar cz build src/wrappers/themis/wasm > build.tgz
+      - name: Upload build directory
+        uses: actions/upload-artifact@v2
+        with:
+          name: build
+          path: build.tgz
+          retention-days: 1 # can be dropped after this build is complete
+
+  unit-tests:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    needs: build-wasmthemis
+    strategy:
+      matrix:
+        node-version:
+          - 8.x   # legacy
```

Comment:
Node 8 is 2019 so I think it makes sense to keep it: https://nodejs.org/dist/latest-v8.x/

Reply:
Node 8 is “legacy” in the sense that it's a maintained LTS release anymore. Node 10 also was EOLed like a month ago.

But at least my personal position on this is to keep testing the old versions and environments as long as it's feasible. So as long as *all* that we need to do to test with Node 8 is to keep this one line:
```
- 8.x
```
Then I'm fine with keeping it around. Users of Node 8 should be well-aware that their environment is deprecated and EOL. If WasmThemis breaks on Node 8 and fixing it is not something easy – so be it, support for it will be retroactively retired. You can think of it as being on a “life support”, rather then “actively supported”.

Reply:
Let's keep Node8